source/_templates/analytics.html

```diff
@@ -1,57 +1,28 @@
-<!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=G-TE81YY48VB"></script>
-<script>
-window.dataLayer = window.dataLayer || [];
-function gtag(){dataLayer.push(arguments);}
-
-gtag('consent', 'default', { 'ad_storage': 'denied', 'analytics_storage': loadChoice() ? 'granted' : 'denied' });
-gtag('js', new Date());
-gtag('config', 'G-TE81YY48VB', {'anonymize_ip': true});
-
-// Save the choice in a cookie
-function storeChoice(granted) {
-    document.cookie = "cookie-consent=" + granted
-}
-
-// Look for a cookie storing the previous choice of the user.
-// Returns undefined if no choice was stored, otherwise a boolean for whether the
-// user gave consent.
-function loadChoice() {
-    if (!document.cookie.includes("cookie-consent=")) {
-        return undefined;
-    }
-    return document.cookie.includes("cookie-consent=true")
-}
-
-// Removes the cookie consent banner
-function removeCookieConsentBanner() {
-    document.getElementById("cookie-consent").remove()
-}
+<!DOCTYPE html>
+<html>
+<meta name="viewport" content="width=device-width, initial-scale=1">
 
-// Called by the cookie consent banner if granted permission
-function consentGranted() {
-    storeChoice(true)
-    gtag('consent', 'update', { 'ad_storage': 'denied', 'analytics_storage': 'granted' });
-    removeCookieConsentBanner()
-}
 
-// Called by the cookie consent banner if denied permission
-function consentDenied() {
-    storeChoice(false)
-    removeCookieConsentBanner()
-}
-
-
-</script>
-<div id="cookie-consent" class="consent">
+<div id="matomo-opt-out" class="consent">
     <p>We use cookies to ensure that we give you the best experience on our website, but only if you grant us the permission.</p>
```

Comment:
I think the message should be changed, since it is now opt-out instead of opt-in